public/player.js

```diff
@@ -282,12 +299,17 @@ async function refreshAudioSources({ reset } = { reset: false }) {
     if (!audioSources[id]) {
       audioSources[id] = {
         segment: chooseSegment(),
-        node: audioContext.createBufferSource()
+        node: audioContext.createBufferSource(),
+        gainNode: audioContext.createGain()
```

Comment:
I suppose that browsers don't collecte nodes that are still connected to the audio graph for garbage, so you should call disconnect on the gain nodes when the buffer source node is removed (line 292).